packages/path-provider/android/src/main/java/io/flutter/plugins/path_provider/PathProviderPlugin.java

```diff
@@ -46,4 +51,8 @@ private String getPathProviderTemporaryDirectory() {
   private String getPathProviderApplicationDocumentsDirectory() {
     return PathUtils.getDataDirectory(activity);
   }
+
+  private String getPathProviderStorageDirectory() {
+    return Environment.getExternalStorageDirectory().getAbsolutePath();
```

Comment:
What is the use case for this? The documentation for [getExternalDirectory()](https://developer.android.com/reference/android/os/Environment.html#getExternalStorageDirectory()) says not to use this directly but instead use [getExternalStoragePublicDirectory(String)](https://developer.android.com/reference/android/os/Environment.html#getExternalStoragePublicDirectory(java.lang.String)) .